partition_snapshot_reader.hh

```diff
@@ -334,7 +334,12 @@ private:
         }
```

Comment:
It would be probably better to move `ck_range_snapshot` and `ck_range_query` definitions to the beginning of `read_next()` and use them everywhere, don't use `*_current_ck_range` directly (like in the `next_row` call above). It will make it clearer which range you pass to `next_row`.